sdk/cosmos/sdk/src/main/java/com/azure/data/cosmos/sync/CosmosSyncClient.java

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.cosmos.sync;
+
+import com.azure.data.cosmos.CosmosClient;
+import com.azure.data.cosmos.CosmosClientBuilder;
+import com.azure.data.cosmos.CosmosClientException;
+import com.azure.data.cosmos.CosmosDatabaseProperties;
+import com.azure.data.cosmos.CosmosDatabaseRequestOptions;
+import com.azure.data.cosmos.CosmosDatabaseResponse;
+import com.azure.data.cosmos.FeedOptions;
+import com.azure.data.cosmos.FeedResponse;
+import com.azure.data.cosmos.SqlQuerySpec;
+import reactor.core.Exceptions;
+import reactor.core.publisher.Mono;
+
+import java.util.Iterator;
+
+/**
+ * Provides a client-side logical representation of the Azure Cosmos database service.
+ * SyncClient is used to perform operations in a synchronous way
+ */
+public class CosmosSyncClient {
```

Comment:
CosmosSyncClient should `implements AutoCloseable` and I see you already have the close() implemented on this, just add the `@Override`

Reply:
Done